docs/body.md

```diff
@@ -768,108 +823,84 @@ For manual configuration, you must modify every Ceilometer node as follows:
 
-            ManagePipeline: true
-            PipelinePublishers:
-              - https://<zenoss_ip>:8342/ceilometer/v1/samples/<zenoss_devicename>?verify_ssl=False
-              - <other_pipeline_publishers>
-            ManageEventPipeline: true
-            EventPipelinePublishers:
-              - https://<zenoss_ip>:8342/ceilometer/v1/events/<zenoss_devicename>?verify_ssl=False
-              - <other_event_pipeline_publishers>
+      [oslo_messaging_notifications]
+      driver = messagingv2
 
-    For example:
+* Configure neutron to send events for ceilometer to forward to zenoss.  Add the following settings to `/etc/neutron/neutron.conf`:
 
-            ManagePipeline: true
-            PipelinePublishers:
-              - https://10.0.0.10:8342/ceilometer/v1/samples/myopenstack?verify_ssl=False
-              - https://10.0.0.12:8342/ceilometer/v1/samples/myopenstack?verify_ssl=False
-            ManageEventPipeline: true
-            EventPipelinePublishers:
-              - https://10.0.0.10:8342/ceilometer/v1/events/myopenstack?verify_ssl=False
-              - https://10.0.0.12:8342/ceilometer/v1/events/myopenstack?verify_ssl=False
+      [oslo_messaging_notifications]
+      driver = messagingv2
+      topics = notifications
 
-*   This template must be ***rendered*** into your templates before you initiate a deployment from
-    Undercloud.<br>
-    For more information on RHOSP template management, see RedHat's
-    [Including Environment Files in Overcloud Creation](https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html-single/advanced_overcloud_customization/index#sect-Including_Environment_Files_in_Overcloud_Creation).
+#### Ceilometer Troubleshooting
 
-*   After deployment is complete, go to the undercloud, SSH into controller and go
-    to `/etc/ceilometer` in ceilometer_agent_notification container to check if
-    `pipeline.yaml` and `event_pipeline.yaml` file is updated:
+A variety of errors can be returned by zenopenstack to ceilometer.  Here are the most common ones.
 
-            ssh heat-admin@<controller ip>
-            sudo su -
-            sudo docker exec --user 0 -it ceilometer_agent_notification /bin/bash
-            # Ensure your configuration had the right publishers
+The errors would be found in `/var/log/ceilometer/agent-notification.log` on the OpenStack controller nodes.
 
-#### VM State Changes
+* Connection Refused
 
-By default, instance state changes will be captured by Zenoss when
-certain events occur, for example, when an instance is shut down, the
-state change to SHUTDOWN will be reflected in Zenoss.
+  ERROR ceilometer.pipeline.sample ConnectionError: HTTPSConnectionPool(host='1.2.3.4', port=8342): Max retries exceeded with url: /ceilometer/v1/samples/myopenstack (Caused by NewConnectionError('<requests.packages.urllib3.connection.VerifiedHTTPSConnection object at 0x7f80d1e89590>: Failed to establish a new connection: [Errno 111] Connection refused',))
```

Comment:
This doesn't format correctly as html.
If indented by 8 chars, it will be valid preformatted text, however, it will be a single line.
I would break the line (in best possible way), and indent by 8 chars.